WordPressKit/WordPressOrgXMLRPCApi.swift

```diff
@@ -208,12 +209,13 @@ open class WordPressOrgXMLRPCApi: NSObject {
                 }.response(queue: DispatchQueue.global()) { (response) in
                     do {
                         let responseObject = try self.handleResponseWithData(response.data, urlResponse: response.response, error: response.error as NSError?)
-                        progress.completedUnitCount = progress.totalUnitCount
                         DispatchQueue.main.async {
+                            progress.completedUnitCount = progress.totalUnitCount
                             success(responseObject, response.response)
                         }
                     } catch let error as NSError {
                         DispatchQueue.main.async {
+                            progress.completedUnitCount = progress.totalUnitCount
```

Comment:
Other changes are all moving an existing code from a background queue to the main queue. This line is new, but I think it's necessary—marking the progress as completed—to be consistent with the success scenario and the `callMethod` function above.

Reply:
A bonus bug fix 😄